fzf-gcloud.plugin.zsh

```diff
@@ -42,13 +42,20 @@ __gcloud_cmd_cache() {
     );
 eof
 
-  gcloud_bin_path="$(which gcloud)"
-  gcloud_root=${gcloud_bin_path%%'/bin/gcloud'}
-  gcloud_docs_root="$gcloud_root/lib/surface"
-  if [ ! -d "$gcloud_docs_root" ]; then
-    echo "Could not find a valid gcloud docs folder at: '$gcloud_bin_path'."
-    return 1
-  fi
+# Determine the path to the gcloud binary, resolving any symlinks with realpath
```

Comment:
nit: would prefer if this were indented at the same level as above, but I can fix that later